apps/admin-ui/src/component/RichTextInput.tsx

```diff
@@ -47,6 +47,7 @@ const ErrorText = styled.div`
   white-space: pre-line;
 `;
 
+// TODO replace with Flex
 const AlignVertically = styled.div`
   display: flex;
   flex-direction: row;
```

Comment:
DO. Or do we even need AlignVertical? or does `align: center` in parent work here?